moderator/moderate/views.py

```diff
@@ -124,9 +127,20 @@ def moderate_event(request, slug, q_id=None, accepted=None):
             question_form = QuestionForm(request.POST, instance=question)
             # update the question with moderator's reply
             if question_form.is_valid():
```

Comment:
If this form isn't valid, then the email isn't sent, without any indication to the moderator. The question also disappears from the queue because its `is_accepted` value has been updated.

I think we should only touch the DB after we've sent the email, that way if it throws an exception the question will stay in the queue and can be re-moderated.

Validation errors and any email-related exceptions should be caught and surfaced to the moderator through the banner messages (I imagine automatically opening the modal again is gonna be a pain).